implementation-ios.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: home
+title: Implementation guide for iOS (Devs)
+nav_order: 6
+nav_titles: true
+titles_max_depth: 2
+---
+This guide is for **application developers** working to add native MPTCP support
+when running on iOS.
+
+## MPTCP connection
+
+iOS offers the possibility to establish new MPTCP connections by using its `URLSession` API, or `NSURLSession` API if you are working with Objective-C. Note that on iOS, only client side MPTCP is supported, and that the path manager can't be manually configured. However, the developers can influence the scheduler of MPTCP by selecting a type of service needed by their app.
+
+ To establish a MPTCP connection, you simply have to tweak a bit the configuration used to create the `(NS)URLSession` object. 
+
+1. Create an `(NS)URLSessionConfiguration` object, typically by using the `default` configuration. 
+2. Select the type of service you want. Available types are:
+
+    | Type of service       | Description         | 
+    |:-------------|:------------------|
+    | None     |The default service type indicating that Multipath TCP should not be used.|
+    | Handover |A Multipath TCP service that provides seamless handover between Wi-Fi and cellular in order to preserve the connection.|
+    | Interactive |A service whereby Multipath TCP attempts to use the lowest-latency interface.|
+    | Aggregate   |A service that aggregates the capacities of other Multipath options in an attempt to increase throughput and minimize latency.|
+
+    {: .note }
```

Comment:
I don't think we need a `note` in blue for that, the following are just links to the official doc.